include/psa/crypto_extra.h

```diff
@@ -224,6 +224,17 @@ void mbedtls_psa_get_stats( mbedtls_psa_stats_t *stats );
 psa_status_t mbedtls_psa_inject_entropy(const unsigned char *seed,
                                         size_t seed_size);
 
+/*
+ * If this option is not turned on, then the function `psa_key_derivation()`
+ * is removed.
+ *
```

Comment:
Minor: commit 683535935 - description typo: `[...] tests regarding key derivation **hasn't** been adapted to [...]` -> `[...] tests regarding key derivation **haven't** been adapted to [...]`